client/main.lua

```diff
@@ -74,225 +72,175 @@ RegisterNetEvent('qb-pawnshop:client:openMenu', function()
             for i = 1, #data.items do
                 if v.name == data.items[i].item then
                     meltMenu[#meltMenu + 1] = {
-                        header = QBCore.Shared.Items[v.name].label,
-                        txt = Lang:t('info.melt_item', { value = QBCore.Shared.Items[v.name].label }),
-                        params = {
-                            event = 'qb-pawnshop:client:meltItems',
-                            args = {
-                                label = QBCore.Shared.Items[v.name].label,
-                                reward = data.items[i].rewards,
-                                name = v.name,
-                                amount = v.amount,
-                                time = data.items[i].meltTime
-                            }
+                        title = QBCore.Shared.Items[v.name].label,
+                        description = Lang:t('info.melt_item', { value = QBCore.Shared.Items[v.name].label }),
+                        event = 'qb-pawnshop:client:meltItems',
+                        args = {
+                            label = QBCore.Shared.Items[v.name].label,
+                            reward = data.items[i].rewards,
+                            name = v.name,
+                            amount = v.amount,
+                            time = data.items[i].meltTime
                         }
                     }
                 end
             end
         end
-        meltMenu[#meltMenu + 1] = {
-            header = Lang:t('info.back'),
-            params = {
-                event = 'qb-pawnshop:client:openMenu'
-            }
-        }
-        exports['qb-menu']:openMenu(meltMenu)
+        lib.registerContext({
+            id = 'open_meltMenu',
+            menu = 'open_pawnShop',
+            title = Lang:t('info.title'),
+            options = meltMenu
+        })
+        lib.showContext('open_meltMenu')
     end)
 end)
 
 RegisterNetEvent('qb-pawnshop:client:pawnitems', function(item)
-    local sellingItem = exports['qb-input']:ShowInput({
-        header = Lang:t('info.title'),
-        submitText = Lang:t('info.sell'),
-        inputs = {
-            {
-                type = 'number',
-                isRequired = false,
-                name = 'amount',
-                text = Lang:t('info.max', { value = item.amount })
-            }
+    local sellingItem = lib.inputDialog(Lang:t('info.title'), {
+        {
+            type = 'number',
+            label = 'amount',
+            placeholder = Lang:t('info.max', { value = item.amount })
         }
     })
     if sellingItem then
-        if not sellingItem.amount then
-            return
-        end
-
-        if tonumber(sellingItem.amount) > 0 then
-            TriggerServerEvent('qb-pawnshop:server:sellPawnItems', item.name, sellingItem.amount, item.price)
-        else
-            QBCore.Functions.Notify(Lang:t('error.negative'), 'error')
-        end
+        if not sellingItem[1] then return end
```

Comment:
```suggestion
        if not sellingItem[1] or sellingItem[1] <= 0 then return end
```
Keep checking for 0 or less